<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="settingsBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>240</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QPushButton" name="settingsHelpButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="delayCheck">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>42</y>
       <width>259</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Reflex mode: delay on falling blocks</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="refillCheck">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>85</y>
       <width>248</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Infinite mode: refill missing blocks</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gridGroupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Grid size</string>
    </property>
    <widget class="QPushButton" name="gridSettingsButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply and reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gridHelpButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gridSizeXLine">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>41</y>
       <width>69</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="x">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>41</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gridSizeYLine">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>41</y>
       <width>70</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>971</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberScore">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>91</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="movementBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>110</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Select two blocks</string>
    </property>
    <widget class="QLabel" name="endLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>64</y>
       <width>25</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QLabel" name="startLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>34</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="endPointLine">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>64</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="startPointLine">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="moveHelpButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="moveButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>813</x>
      <y>11</y>
      <width>85</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>904</x>
      <y>11</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMins">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>41</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>41</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberSecs">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>41</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
